apps/ckit-app/src/components/Wallet/WalletChange.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button, Col, Empty, Row } from 'antd';
+import { observer } from 'mobx-react-lite';
+import React from 'react';
+import { useSigner, WalletContainer } from 'containers/WalletContainer';
+
+const NervosExploreUrl = 'https://explorer.nervos.org/aggron/address/';
+
+export const WalletChange = observer(() => {
+  const { wallets, currentWalletIndex, setModalVisible } = WalletContainer.useContainer();
+  const wallet = currentWalletIndex === null ? undefined : wallets[currentWalletIndex];
```

Comment:
this can be refactored to 

```tsx
// WalletContainer.ts
const selectedWallet= useMemo<Wallet | undefined>(() => {
  return 
}, [wallets, currentWalletIndex])

return {
...
selectedWallet
}
```

```tsx
// WalletChange.tsx

const { selectedWallet } = WalletContainer.useContainer();
...
```

Reply:
fixed